airflow/providers/weaviate/hooks/weaviate.py

```diff
@@ -315,6 +315,88 @@ def batch_data(
                         )
                         self.log.debug("Inserted object with uuid: %s into batch", uuid)
 
+    def batch_create_links(
+        self,
+        data: list[dict[str, Any]] | pd.DataFrame,
+        collection_name: str = "",
+        batch_size: int = 100,
+        concurrent_requests: int = 10,
+        consistency_level: Any = None,
+        retry_attempts_per_object: int = 5,
+    ):
+        """
+        Batch create links from an object to another object through cross-references.
+
+        :param collection_name: The name of the collection that objects belongs to.
+        :param data: list or dataframe of objects we want to add.
+        :param batch_size: The number of objects to batch together.
+        :param concurrent_requests: The number of concurrent requests to make.
+        :param consistency_level: The consistency level to use.
+        :param creation_time: The creation time to use.
+        :param retry_attempts_per_object: number of time to try in case of failure before giving up..
+        """
+        client = self.get_conn()
+        collection_name_given = collection_name
+        converted_data = self._convert_dataframe_to_list(data)
+
+        for data_obj in converted_data:
+            if collection_name == "":
+                collection_name = data_obj.pop("collection_name", None)
+                if collection_name is None:
+                    raise ValueError("collection_name is a required field")
+                elif not isinstance(collection_name, str) or collection_name == "":
+                    raise ValueError("collection_name should be a valid string")
```

Comment:
technically "" is a valid string 🤔 a non-empty string maybe?
```suggestion
                    raise ValueError("collection_name should be a non-empty string")
```